packages/accounts/src/msca/plugins/multisig/index.ts

```diff
@@ -13,4 +13,7 @@ export {
 } from "./plugin.js";
 export type * from "./types.js";
 export { type SignerType } from "./types.js";
-export { formatSignatures, getSignerType } from "./utils.js";
+export { combineSignatures } from "./utils/combineSignatures.js";
+export { formatSignatures } from "./utils/formatSignatures.js";
+export { getSignerType } from "./utils/getSignerType.js";
+export { splitAggregatedSignature } from "./utils/splitAggregatedSignature.js";
```

Comment:
ah these also need to be exported from `accounts/src/index` as well so that consumers can access them

Reply:
nit: would also export them to a `./utils/index` file first so you can do something like:

```suggestion
export { combineSignatures, formatSignatures, getSignerType, splitAggregatedSignature } from "./utils/index.js";
```